### ISSUE TITLE
Error Train bisenet on My Own Data

### ISSUE THREAD

**Issue by HienTran1997** [pdoublerainbow/bisenet-tensorflow#11]
I want to train on my own Dataset (100 files png for test train road segmentation). I have split my data and create file.csv like CamVid, edit the num_classes on train.py to 1. When I train, from the step 200 above, the loss became too large and then it goes to "NaN", and the predict_loss still 0.00
Does Anyone know what is the problem. Is the problem from the parametters on files configuration.py and how can I fix it ?
